ingest/tests/common/mod.rs

```diff
@@ -164,6 +178,32 @@ impl TestClient {
             hex_usage_stats_file_sink_rx,
             radio_usage_stats_file_sink_rx,
             unique_connections_file_sink_rx,
+            cell_heartbeat_rx,
+            data_transfer_rx,
+        }
+    }
+
+    pub async fn cell_heartbeat_recv(mut self) -> anyhow::Result<CellHeartbeatIngestReportV1> {
+        match timeout(Duration::from_secs(2), self.cell_heartbeat_rx.recv()).await {
```

Comment:
What’s the timeout for?

Reply:
The message should be in the channel very quickly, so didn't want the test to take longer than 2 seconds to fail